options/train_options_regression.py

```diff
@@ -0,0 +1,168 @@
+import argparse
+import os
+from util import util
+import torch
+import models
+import data
+
+
+class TrainOptionsRegression():
```

Comment:
Not sure why we have a brand new class here that has all options from other existing classes just because the semantic criterion changes.

Reply:
Also, most options are not related to a simple regression model, right ?